client/src/components/Layout/Login/Login.js

```diff
@@ -63,11 +65,11 @@ class Login extends Component {
           >
             Login
           </button>
-          <Link to="/" className="text-link">
-            <button type="submit" className="login-form__back__btn">
-              Back
+
+            <button type="submit" className="login-form__back__btn" onClick={ () => {
+          this.goBack()
+        }}> Back
```

Comment:
You can remove the `goBack` function on line 34 and here just write `onClick={this.props.history.goBack}` (you're passing a reference to the `props.history.goBack` function to `onClick`, so when `onClick` is invoked, it will invoke the function it is pointing to, which is the `goBack`).